media/src/androidMain/kotlin/DefaultMediaManager.kt

```
@@ -136,6 +138,11 @@ actual class DefaultMediaManager(mediaSurfaceProvider: MediaSurfaceProvider?, co
             } else {
                 mediaPlayer.setDataSource(source.context, source.uri, source.headers)
             }
+            is MediaSource.Bundle -> {
+                ApplicationHolder.applicationContext.let { context ->
```

`ApplicationHolder.applicationContext` should normally not be referenced directly like this, but as a default value for an argument. That also avoids this silent failure you have here.

 If `expect` requires it, you can also have a `actual` secondary constructor use it directly to call the primary one (and error if it's null)

---

Fixed